.github/workflows/ci.yml

```diff
@@ -126,4 +132,6 @@ jobs:
       - name: Run e2e tests
         run: |
           export PATH=$HOME/.cargo/bin:$HOME/.local/bin:$HOME/go/bin:$PATH
-          make e2e_only
+          cd api_tests
```

Comment:
Can you explain what is stopping you to use the makefile?

Not using the makefile in the CI can lead to unwanted discrepancies between local and CI development. This already bit us back when we would run `cargo test --doc` locally but not in the CI (or the opposite?).